fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -78,10 +79,15 @@ def segment_FOV(
     """
     Internal function that runs Cellpose segmentation for a single ROI.
 
-    :param column: Three-dimensional numpy array
+    :param x: numpy array
```

Comment:
We should document here what is the expected shape
(always 4D, right?)

Reply:
Yes, now that it's always 4D, that's true :)

Reply:
I added that in now